docs/pages/api-reference/connectors/pubsub.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Pub/Sub
+order: 0
+status: published
+---
+### Pub/Sub
+Data connector to Google Pub/Sub messaging service.
+
+#### Project Parameters
+<Expandable title="name" type="str">
+A name to identify the source. The name should be unique across all Fennel sources.
+</Expandable>
+
+<Expandable title="project_id" type="str">
+The project ID of the Google Cloud project containing the Pub/Sub topic
+</Expandable>
+
+<Expandable title="service_account_key" type="str">
+The JSON string containing the credentials for the Service Account to use to access
+Pub/Sub topic. See below for instructions on how to obtain this.
+</Expandable>
+
+:::info
+Fennel supports only Upsert mode CDC with data in JSON format. If you require support
+for schema or CDC data format, please reach out to Fennel support.
+:::
+
+#### Topic Parameters
+<Expandable title="topic_id" type="str">
```

Comment:
Is `topic_id` a standard GCP thing? If not, let's just call it `name` since `id` usually denotes an integer, not a string.

Reply:
yeah its a pub-sub specific thing - 
![Uploading image.png…]()

Reply:
Yes that is the standard naming convention